src/game/handlers/inventory.js

```diff
@@ -55,21 +60,21 @@ export default function call (dao, provider, _, msg) {
             ).join('\n') || _('Nothing.'),
         ),
 
-        sortBy(x => findIndex(equals(x[0]), ["weapon", "set", "token"]),
-              toPairs(groupBy(prop("type"), equips)))
-          .map(group =>
-            [
-              _(`<pre>${capitalize(group[0])}:</pre>`),
-              group[1]
-                .sort(comparator((x, y) => x.tier < y.tier))
-                .map(eq =>
-                  `<b>${eq.name}</b> - T${eq.tier} - ` +
-                  `/${equip_verbs.find(x => x.type == eq.type).verb}_${eq.id}`
-                ).join('\n')
-            ].join('\n'),
-          ).join('\n\n'),
+        join('\n\n', map((pair, equipType = pair[0], equipList = pair[1]) =>
+          join('\n', [
+            _(`<pre>${capitalize(equipType)}:</pre>`),
+            ...map(eq =>
+              `<b>${eq.name}</b> - T${eq.tier} - ` +
```

Comment:
pro tip: since you are using a lot of eq.stuff, you could get these stuff using destructuring to make the code better.

```
const { name, tier, type, id } = eq // 'eq' could be 'equip' as well since there is no need to repeat it a lot :)
```